### ISSUE TITLE
Sample bug

### ISSUE THREAD

**Issue by edsantiago** [edsantiago/greasemonkey#1]
At the time I am creating this, the repo is prepopulated with eight labels, one of which is the lone word 'bug'. I am attaching that label before pressing submit.

**Comment by edsantiago** [comment]
Two queries, the first one `bug`, the second `kind/bug`:
```console
$ curl -H "Authorization: bearer $GITHUB_TOKEN" -X POST --data '{ "query": "{ search(query: \"repo:edsantiago/greasemonkey is:issue is:open label:bug\", type:ISSUE) { issueCount } } " }' https://api.github.com/graphql
{"data":{"search":{"issueCount":1}}}
```
```console
$ curl -H "Authorization: bearer $GITHUB_TOKEN" -X POST --data '{ "query": "{ search(query: \"repo:edsantiago/greasemonkey is:issue is:open label:kind/bug\", type:ISSUE) { issueCount } } " }' https://api.github.com/graphql
{"data":{"search":{"issueCount":0}}}
```
Results exactly as expected.

**Comment by edsantiago** [comment]
Clicked the gear next to `Labels`. Menu, -> [pencil] Edit labels.  Changed the name "bugs" to "kind/bugs". It was easy.

Verified that the label on this issue is now "kind/bug", same color. Now the query:
```
{"data":{"search":{"issueCount":0}}}   -- bug
{"data":{"search":{"issueCount":1}}}   -- kind/bug
```
Again as expected.

**Comment by edsantiago** [comment]
Quick note: immediately below the opening comment, github says "edsantiago added the kind/bug label". At the time I created the bug, and before I did the rename, the red text read just "bug". It seems (and this is reasonable) that the labels are indexed into the label DB; they are not displayed as fixed strings in page. So, in a given repo, upon renaming a label, all uses of that label will be changed. (But any human-text-written references will not, obviously).
